scripts/git_version_for_docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
`/usr/bin/env` was used in previous script, we can use it here too to be more consistent.